package.json

```diff
@@ -64,9 +64,8 @@
     "screwdriver-data-schema": "^12.0.0",
     "screwdriver-datastore-dynamodb": "^2.1.0",
     "screwdriver-datastore-imdb": "^1.0.0",
-    "screwdriver-executor-k8s": "^7.0.0",
-    "screwdriver-executor-l3l": "^1.0.0",
-    "screwdriver-models": "^10.3.0",
+    "screwdriver-executor-k8s": "^8.0.0",
+    "screwdriver-models": "^11.0.0",
```

Comment:
Because there isn't one?  I'm just bumping to 11 in this PR: https://github.com/screwdriver-cd/models/pull/104